client/src/components/App.jsx

```diff
@@ -25,37 +25,33 @@ class App extends React.Component {
   }
 
   handleGetClickInfo (e) {
-    var arrayOfWidgets = ['review-container', 'overview', 'RICWid', 'header'];
     var time = moment().format().toString();
     var widgetName;
-
+    var elementType = e.target.nodeName;
     checkClasses(e.target);
+    Atelier.logClick(elementType, widgetName, time);
 
     function checkClasses (element) {
```

Comment:
Also, base on the multiple if/else statements, I'm think this is a case where you can use switch, but only a suggestion, don't need to actually utilize

Reply:
Yeah I was thinking about changing it to a switch statement but wanted to just get it working for now. Will work on implementing the change.